packages/arb-ts/src/lib/l2Bridge.ts

```diff
@@ -118,75 +135,43 @@ export class L2Bridge {
     ](erc20l1Address, to, amount, '0x', overrides)
   }
 
-  public async updateAllL2Tokens(): Promise<Tokens> {
-    for (const l1Address in this.l2Tokens) {
-      const l2Address = this.l2Tokens[l1Address]?.ERC20?.contract.address
-      if (l2Address) {
-        await this.getAndUpdateL2TokenData(l1Address, l2Address)
-      }
-    }
-    return this.l2Tokens
+  public async contractExists(contractAddress: string): Promise<boolean> {
+    const contractCode = await this.l2Provider.getCode(contractAddress)
+    return !(contractCode.length > 2)
```

Comment:
I might be missing a JS thing here, but doesn't this return true if length is less than or equal to 0?

Reply:
this function was removed since its not needed anymore